src/torrent.cpp

```diff
@@ -12253,7 +12269,7 @@ namespace libtorrent
 		TORRENT_ASSERT(is_single_thread());
 		INVARIANT_CHECK;
 
-		return (std::max)(0, int(m_connections.size()) - m_num_seeds - m_num_connecting);
```

Comment:
and perhaps that ``std::max`` statement can be removed now, replaced by an assert